app/routes/__private/user/employees.$uuid.tsx

```diff
@@ -9,16 +9,35 @@ import remixAppAction from "~/data/remixAppAction.server";
 import remixAppLoader from "~/data/remixAppLoader.server";
 
 const EmployeeProfilePage = () => {
-  const { employee, responsibilities } =
+  const employee =
     useLoaderData<Awaited<ReturnType<typeof getUserEmployeeProfile>>>();
   const fireEmployeeRef = useRef<DialogRef>(null);
   return (
     <div className="flex flex-col h-full">
       <h1 className="text-3xl font-semibold mb-4">{employee.title}</h1>
-      <h3 className="text-md italic mb-16">{employee.instanceId}</h3>
+      <div className="mb-16 border-dashed border p-4">
+        <h3 className="text-lg font-bold mb-4">Employee Device Info</h3>
+        <p className="italic mb-2">
+          {employee.instance.id} - {employee.instance.state}
+        </p>
+        <div className="flex justify-between items-start">
+          <code>
+            ssh -i {employee.uuid}.pem {employee.instance.username}@
+            {employee.instance.dnsName}
+          </code>
+          <Link
+            to="key"
+            download={`${employee.uuid}.pem`}
+            className="underline text-sky-500"
+            reloadDocument
+          >
+            Download Private Key
+          </Link>
+        </div>
+      </div>
       <div className="flex-grow">
         <h2 className="text-2xl">Responsibilities</h2>
-        {responsibilities.map((r) => (
+        {employee.responsibilities.map((r) => (
```

Comment:
deferring this for now.

Devin has a notion of "Tips" which is kind of cool. However, I realized through that experience that I didn't want to enter it in Devin's UI